content/12.privacy.md

```diff
@@ -7,13 +7,14 @@ Therefore, it is important to think thoroughly and cautiously about deep learnin
-Teams can institute what Rachel Thomas of the Center for Applied Data Ethics at the University of San Francisco terms “ethical risk sweeps.”
-Teams focus on “periodically scheduling times to really go through what could go wrong and what are the ethical risks.”
-After all, “ethics is thinking through what can go wrong before it does and having processes in place around what happens when there are mistakes or errors” [@https://venturebeat.com/2019/10/07/how-to-operationalize-ai-ethics/].
-Second, to help foster a conscious ethics-oriented mindset, researchers should consider expanding journal clubs to include scholarly and popular articles detailing real-world ethics issues relevant to in their scientific fields.
-This will help researchers to think more holistically and judiciously (that is, with good judgment and sense) about their work and its implications.
-Third, we encourage individual- and team-level participation in professional societies and other types of organizations and initiatives related to AI/deep learning and data ethics [@https://ocean.sagepub.com/blog/10-organizations-leading-the-way-in-ethical-ai; @https://www.aies-conference.com/2021/; @http://ethics-artificial-intelligence-conference.mozello.com].
+First, just as it is a best practive to keep a project-specific or programming-related issue tracker detailing known bugs and other technical issues, practitioners should get into the habit of keeping an active _ethics register_.
+In this register, ethical concerns can be raised, recorded, and resolved, exactly as software problems are triaged and fixed. 
+Because projects using deep learning usually rely on writing code, an ethics register can be a part of the issue tracker in the version-control system for the software itself.
+By colocating the two, practioners can operationalize the concept that ethical problems arer "bugs" that must be resolved, not nice-to-haves that can be considered at some indefinite point in the future.
+For practitioners intending to distribute trained models, having an ethics register can also facilitate creating a *model card* [@doi:10.1145/3287560.3287596], a short document specifying the domains in which the model's performance was validated (for example, which model organism was used) and how the performance was benchmarked, along with known limitations and concerns.
+Second, to help foster a conscious ethics-oriented mindset, researchers should consider expanding journal clubs to include scholarly and popular articles detailing real-world ethics issues relevant to their scientific fields.
+This will help researchers to think more holistically and judiciously about their work and its implications.
+Third, we encourage individual- and team-level participation in professional societies and other types of organizations and initiatives related to the domains of AI and data ethics as well as bioethics.
```

Comment:
I liked having a few examples here but don't feel strongly about it.

Reply:
Ok I added them back albeit with more stable identifiers